site-book/bin/generate-md.sh

```diff
@@ -190,7 +190,13 @@ cd site-book/src/site/markdown
 # Rewrite hrefs for resource references, using table provided by Maintainers
 for (( i=0; i<${#HREF_REWRITE_LIST[@]} ; i+=2 )) ; do
     echo rewriting href in "${HREF_REWRITE_LIST[$i]}" : "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}"
-    sed -i '' -e "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}" "${HREF_REWRITE_LIST[$i]}"
+    if [[ "$OSTYPE" =~ "darwin" ]] ; then
+        # Macs need an argument after -i option 
+        sed -i '' -e "${HREF_REWRITE_LIST[ $(( i + 1 )) ]}" "${HREF_REWRITE_LIST[$i]}"
+    else
```

Comment:
@JonZeolla thanks for the example pointer.  Fixed.

@james-sirota , yes maybe sorta.  But that's work too, and the on-going nuisance for people trying to use it isn't adequately balanced for the effort.  I'm almost done with the conversion to python.